### ISSUE TITLE
mypingponganalysis is broken due to faulty method call

### ISSUE THREAD

**Issue by boegel** [hpcugent/mympingpong#18]
```
Traceback (most recent call last):
  File "/software/mympingpong/0.7.1-intel-2016b-Python-2.7.12-TEST/bin/mympingponganalysis", line 4, in <module>
    __import__('pkg_resources').run_script('mympingpong==0.7.1', 'mympingponganalysis')
  File "build/bdist.linux-x86_64/egg/pkg_resources/__init__.py", line 719, in run_script
  File "build/bdist.linux-x86_64/egg/pkg_resources/__init__.py", line 1511, in run_script
  File "/software/mympingpong/0.7.1-intel-2016b-Python-2.7.12-TEST/lib/python2.7/site-packages/mympingpong-0.7.1-py2.7.egg/EGG-INFO/scripts/mympingponganalysis", line 310, in <module>

  File "/user/home/gent/vsc400/vsc40023/eb_phanpyscratch/CO7/sandybridge/software/mympingpong/0.7.1-intel-2016b-Python-2.7.12-TEST/lib/python2.7/site-packages/mympingpong-0.7.1-py2.7.egg/EGG-INFO/scripts/mympingponganalysis", line 259, in plot

TypeError: addtext() takes exactly 3 arguments (4 given)
```

**Comment by boegel** [comment]
fixed with:

```diff
diff --git a/bin/mympingponganalysis.py b/bin/mympingponganalysis.py
index 38263eb..60bbe9c 100755
--- a/bin/mympingponganalysis.py
+++ b/bin/mympingponganalysis.py
@@ -256,7 +256,7 @@ class PingPongAnalysis(object):
         gs1 = gridspec.GridSpec(10, 10, left=0.02, bottom=0.02, right=0.98, top=0.98, wspace=0.05, hspace=0.6)
         vextrema = self.addlatency(self.data, plt.subplot(gs1[0:8, 0:5]), fig1)
         coloredges = self.addglobalhistogram(self.data, plt.subplot(gs1[8:10, 0:4]), vextrema)
-        self.addtext(self.meta, plt.subplot(gs1[0:1, 5:10]), fig1)
+        self.addtext(self.meta, plt.subplot(gs1[0:1, 5:10]))
         self.addsamplesize(self.count, plt.subplot(gs1[1:4, 5:7]), fig1)
         self.addconsistency(self.consistency, plt.subplot(gs1[1:4, 7:9]), fig1)
         if self.latencymask != INTERVAL_NONE:
```

**Comment by boegel** [comment]
fixed in #21
